src/services/api/index.tsx

```diff
@@ -125,7 +125,7 @@ const getDefaultSuggestionQueryBody = (model) => (textInput) => {
     neighbors: 1,
   };
 
-  if (model.inherits.includes('Ontology') || model.name === 'Ontology') {
+  if (model && model.name && schemaDefn.ancestors(model.name) && schemaDefn.ancestors(model.name).includes('Ontology') || model.name === 'Ontology') {
```

Comment:
non-blocking: this condition could be simplified to 
`(model?.name &&  schemaDefn.ancestors(model.name).includes('Ontology')) || model?.name === 'Ontology'`

or even use the `inheritsFrom` method to simplify further
`(model?.name &&  schemaDefn.inheritsFrom(model.name, 'Ontology')) || model?.name === 'Ontology'`